aries_cloudagent/protocols/coordinate_mediation/v1_0/handlers/tests/test_mediation_request_handler.py

```diff
@@ -6,7 +6,7 @@
 from ......messaging.request_context import RequestContext
 from ......messaging.responder import MockResponder
 
-from .. import mediator_request_handler as test_module
+from .. import mediate_request_handler as test_module
```

Comment:
Same type should be camel case thing here.

Reply:
I will change test_module to TestModule.